### DISCUSSION TITLE
Chinese resources from baidu for SSD201 SSD202

### DISCUSSION THREAD

**Discussion by Vasili-Sk** [linux-chenxing/linux-chenxing.org#27] [General]
I tried to collect all stuff that i cud find in web about SSD201 SSD202.
Delete if unwanted that here.

7z archive, should be valid for month:
https://ufile.io/56pl21qh

Useful links:
http://doc.industio.com/docs/ssd201-sbc2d07/ssd201-sbc2d07-1ck3lg2f8uhak
https://item.taobao.com/item.htm?id=629244393880#detail
https://github.com/linux-chenxing/linux-chenxing.org/blob/master/infinity2/SSD201_pb_S_v01.pdf
http://linux-chenxing.org
http://doc.industio.com/docs/ssd20x-system/page_14
https://whycan.com/t_5424.html

**Comment by fifteenhex** [comment]
Is the 7z archive the SDK? We have the u-boot and kernel from it. :)

I think I have links to the other stuff on the wiki. If there is anything missing feel free to add it and create a PR.

**Reply by Vasili-Sk** [reply]
Yes i guess that is SDK, 
it is for those who wants to see it but cant acces to weird baidu disk.

**Comment by Michel-FK** [comment]
Trying to access the 7z archive, I get:

> Premium Access Only
The hosting period for this file has now expired, only paid users can download it.
To download this file, you must first subscribe to a pid plan
File retrieval required
This file has been archived after a prolonged period of inactivity, subscribe to a plan to request a retrieval and we will email you when the file is ready.
This typically takes less than 24 hours, but can take a maximum of 48 hours.

**Reply by Vasili-Sk** [reply]
its outdated, check wiki, it have some links
